src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -17,14 +17,24 @@ namespace Microsoft.NET.Build.Tasks
 {
     public class PrepareForReadyToRunCompilation : TaskBase
     {
+        internal struct CrossgenToolInfo
```

Comment:
Looks like all uses of this are private, so this can be private. 

nit: consider moving this lower, away from the public task parameters.